src/entity/transactions/payout/payout-request.ts

```diff
@@ -23,13 +23,14 @@ import User from '../../user/user';
 // eslint-disable-next-line import/no-cycle
 import PayoutRequestStatus from './payout-request-status';
 import PayoutRequestPdf from '../../file/payout-request-pdf';
-import { hashJSON } from '../../../helpers/hash';
-import PayoutRequestPdfService from '../../../service/payout-request-pdf-service';
 import BasePayout from './base-payout';
+import { PdfAble } from '../../file/pdf-able';
+import PayoutRequestPdfService from '../../../service/pdf/payout-request-pdf-service';
+import { PAYOUT_REQUEST_PDF_LOCATION } from '../../../files/storage';
 
 @Entity()
 // TODO Migration
```

Comment:
Officially not part of this PR, but if this TODO is resolved, please remove it here as well :)